client/src/pages/dashboard/Dashboard.js

```diff
@@ -288,22 +256,21 @@ class Dashboard extends Component {
           </Col>
           <Col size="md-2"></Col>
         </Row>
-
         <Row>
           <Col size="md-2"></Col>
           <Col size="md-8">
             <form action="POST">
               <Input
-                // className="new-pitch"
+                name="title"
                 placeholder="Title of project!"
                 onChange={this.handleChange}
                 value={this.state.title}
               />
               <TextArea
-                // className="new-pitch"
+                name="description"
                 style={{ height: "125px" }}
                 placeholder="Describe your project!"
-                onChange={this.handleChange2}
+                onChange={this.handleChange}
```

Comment:
Using just one handleChange function, it uses "name" and "value". We need to use "name" for out input and text area.

Also, even if we stick with two handleChange functions, it's very bad practice to name it "handleChange2".

Reply:
I concur.